app/assets/javascripts/admin/datastore_health_check.js

```diff
@@ -26,11 +26,38 @@ export const pingDataStoreIfAppropriate = memoizedThrottle(async (requestedUrl:
   if (usedDataStore != null) {
     const { url } = usedDataStore;
     const healthEndpoint = `${url}/data/health`;
-    Request.triggerRequest(healthEndpoint, { doNotCatch: true, mode: "cors", timeout: 5000 }).catch(
-      () => {
+    Request.triggerRequest(healthEndpoint, { doNotInvestigate: true, mode: "cors", timeout: 5000 })
+      .then(async () => {
+        const buildinfoWebknossos = await RestAPI.getBuildInfo();
+        const webknossosVersion = buildinfoWebknossos.webknossos.version;
+        const oldestSupportedDatastoreVersion =
+          buildinfoWebknossos.webknossos.oldestSupportedDatastoreVersion;
+
+        const buildinfoDatastore = await RestAPI.getDataStoreBuildInfo(url);
+        const buildInfoWebknossosDatastore = buildinfoDatastore.webknossosDatastore
+          ? buildinfoDatastore.webknossosDatastore
+          : buildinfoDatastore.webknossos;
```

Comment:
When we have a non-standalone-datastore. In this case both requests (for webknossos and datastore) are send to e.g. `localhost:9000/api/buildinfo`, but the buildinfo does not contain a 'webknossosDatastore' because webknossos and the datastore have always the same version (in this case). That is why we look at `webknossos.version`